controllers/project_controller_test.go

```diff
@@ -347,11 +341,11 @@ var _ = Describe("ProjectController", func() {
 					})
 
 					It("allows the user specified in the project access to the project", func() {
-						_, err := reconciler.Reconcile(Request(project.Namespace, project.Name))
+						_, err := reconciler.Reconcile(ctx, Request(project.Namespace, project.Name))
 						Expect(err).NotTo(HaveOccurred())
 
 						clusterRole := &rbacv1.ClusterRoleBinding{}
```

Comment:
Super minor: `clusterRole` could be named `clusterRoleBinding` for clarity. Same with `roleBinding` [here](https://github.com/vmware-tanzu/projects-operator/pull/133/files#diff-aef4ba6209ab52586d16ae6ee1a2659fd3bc7a72bea2cec16a65f73ece9ce966R390).

Reply:
This PR is mainly concerned with updating the dependencies for security reasons, I want to avoid code maintenance even it's as minor as renaming